new(content): added Falco 0.38.1 blog post. (#1330)
falcosecurity/falco-website: content/en/blog/falco-0-38-1/index.md

---
title: Introducing Falco 0.38.1
date: 2024-06-19
author: Federico Di Pierro
slug: falco-0-38-1
tags: ["Falco","Release"]
---

Today we announce the release of **Falco 0.38.1** 🦅!

## Fixes

> Falco's 0.38.1 is a patch release aimed at addressing a few important bugs. It includes the following:

Reviewer: ```suggestion
Falco's 0.38.1 is a patch release aimed at addressing a few important bugs. It includes the following fixes:
```

* **A Falco crash while running with plugins and metrics enabled has been solved** (https://github.com/falcosecurity/falco/issues/3229)

> * Falco `-p` output format option can be passed to plugin events while `-pc` and `-pk` can only be used for syscall sources (https://github.com/falcosecurity/falco/pull/3239)

Reviewer: ```suggestion
* Falco `-p` output format option can now be passed to plugin events while `-pc` and `-pk` can only be used for syscall sources. Fixes an issue that could result in Falco exiting with `LOAD_ERR_COMPILE_OUTPUT` on startup with k8s clusters that had `-pk` and audit enabled (https://github.com/falcosecurity/falco/pull/3239)
```

> * Fixed signed compare operator in libs (https://github.com/falcosecurity/falco/issues/3245)

Reviewer: ```suggestion
* Fixed an issue that could prevent the integer compare operators `<`, `<=`, `>`, `>=` in rules from working properly (https://github.com/falcosecurity/falco/issues/3245)
```
